### DISCUSSION TITLE
Option to Reconnect to SSH Session that has Exited

### DISCUSSION THREAD

**Discussion by niemeier23** [microsoft/terminal#14781] [Ideas]
A situation I find myself in most mornings is waking up my work computer and seeing the WT, with yesterday's SSH sessions (in their own respective profile tabs) having exited ([process exited with code 255 (0x000000ff)], as you may recognize.  (This is, the tab/profile's commandline is an "ssh ...." statement.)

![image](https://user-images.githubusercontent.com/13784308/216638122-6c85d9ca-1e2b-4c4e-a0ee-9505f317383d.png)

It would be very useful to have the WT recognize that state for these tabs that are ssh sessions, and have an option appear to reconnect within the same tab.  To my knowledge, this tab is now useless, except to perhaps scroll up and review output history.  I'm not sure what other caveats or limitations tabs in this state are under, but I think this could be doable and would undoubtedly streamline workflows.

I realize this is a lazy request as I can easily open a new profile tab and close the existing, exited one fairly quickly.  But, it also seems like a waste since I'm already there, in the tab I want to begin (re)using.

**Comment by zadjii-msft** [comment]
Hey wadda ya know, this is something we did in https://github.com/microsoft/terminal/releases/tag/v1.17.1023! See #14060 where this was implemented. 

If this tab's commandline is the `ssh` commandline you want to re-run, then in 1.17, you can just hit <kbd>enter</kbd> and we'll re-run the commandline in that pane. 

That sound like what you're looking for?
